Model/ApiPayloadRequest.php

```diff
@@ -63,6 +63,15 @@ public function __construct($request, Transport $transport, TokenHelper $tokenHe
      */
     public function send()
     {
+        // If it's in test mode, add test_values to TokenHelper.
+        if ($this->test && false === is_null($this->request) && false === is_null($this->request->body)) {
```

Comment:
Let's go farther with this.

null !== $this->request & false !== $this->request->body

Reply:
And at that point, might as well check for the actual value of test, rather than let PHP type juggle